src/app/components/PriceChart.tsx

```diff
@@ -70,37 +106,100 @@ function PriceChartCanvas(props: PriceChartProps) {
 
       return () => {
         window.removeEventListener("resize", handleResize);
-        // clearInterval(intervalId);
         chart.remove();
       };
     }
-  }, [data]);
-
-  return <div ref={chartContainerRef} />;
+  }, [data, dispatch]);
+  //Temporary brute force approach to trim the top of the chart to remove the gap
+  return <div ref={chartContainerRef} className="relative mt-[-1.7rem]"></div>;
 }
 
 export function PriceChart() {
   const state = useAppSelector((state) => state.priceChart);
   const dispatch = useAppDispatch();
+  const candlePeriod = useAppSelector((state) => state.priceChart.candlePeriod);
+  const candlePrice = useAppSelector(
+    (state) => state.priceChart.legendCandlePrice
+  );
+  const change = useAppSelector((state) => state.priceChart.legendChange);
+  const percChange = useAppSelector(
+    (state) => state.priceChart.legendPercChange
+  );
+  const currentVolume = useAppSelector(
+    (state) => state.priceChart.legendCurrentVolume
+  );
+  const isNegativeOrZero = useAppSelector(
+    (state) => state.priceChart.isNegativeOrZero
+  );
 
   return (
     <div>
-      <label htmlFor="candle-period-selector">Candle Period:</label>
-      <select
-        className="select select-ghost"
-        id="candle-period-selector"
-        value={state.candlePeriod}
-        onChange={(e) => {
-          dispatch(setCandlePeriod(e.target.value));
-        }}
-      >
-        {CANDLE_PERIODS.map((period) => (
-          <option key={period} value={period}>
-            {period}
-          </option>
-        ))}
-      </select>
-
+      <div className="">
+        <div className="flex p-[1vh]">
+          {CANDLE_PERIODS.map((period) => (
+            <button
+              key={period}
+              className={`px-[0.5vw] py-[0.5vw] text-sm font-roboto text-#d4e7df hover:bg-white hover:bg-opacity-30 hover:rounded-md ${
+                candlePeriod === period ? "text-blue-500" : ""
+              }`}
+              onClick={() => dispatch(setCandlePeriod(period))}
+            >
+              {period}
+            </button>
+          ))}
+        </div>
+        <div className="flex justify-between text-sm font-roboto">
+          <div className="ml-4">
+            Open:{" "}
+            <span
+              className={isNegativeOrZero ? "text-red-500" : "text-green-500"}
+            >
+              {candlePrice?.open}
+            </span>
+          </div>
+          <div>
+            High:{" "}
+            <span
+              className={isNegativeOrZero ? "text-red-500" : "text-green-500"}
```

Comment:
Here and below (and you probably can specify `{isNegativeOrZero ? ...}` one time in the enclosing div (the one with the `<div className="flex justify-between`) also please use the theme colors to make page look consistent.

Please also remove `font-roboto` I think it should be set up for the whole app.

Reply:
I removed the font-roboto and swapped colors where I could  in https://github.com/DeXter-on-Radix/website/pull/145